aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -1321,4 +1322,157 @@ Tensor any_sparse(const Tensor& self) {
   return at::any(self._values());
 }
 
+Tensor bmm_sparse_cpu(const SparseTensor& self, const Tensor& mat2) {
+  Tensor result = at::empty({}, mat2.options());
+  return bmm_out_sparse_cpu(result, self, mat2);
+}
+
+// Search a sorted strided array for the rightmost instance of a value.
```

Comment:
This comment is a little confusing; what if the value doesn't exist?

Reply:
A quick peek in the contents reveals that you return -1 if it is not found. What if `scalar_t` is an unsigned type? Answer: you only use this function at one specialization, `int64_t`.

Reply:
I'm not actually checking for -1 anywhere though, I just figured I should assign it something consistent, rather than just returning garbage. The `found` bool reference is what the caller checks. I'll amend the comment to mention that.